### PR TITLE
Add single_version_file option

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/556

@GuillaumeGomez, @sdroege Please check texts too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,20 @@ You now have the sys part of your binding!
...
+[options]
+girs_dir = "gir-files"
+library = "Gtk"
+version = "3.0"
+min_cfg_version = "3.4"
+target_path = "."
+work_mode = "normal"
+generate_safety_asserts = true
+deprecate_by_min_version = true
+# with this option enabled, versions for gir and gir-files saved only to one file to minimize noise
```
Comment (GuillaumeGomez) [inline README.md]: Oh so we use it as a non-default option?

File: src/codegen/general.rs [hunk 2]
```diff
@@ -24,6 +27,29 @@ pub fn start_comments(w: &mut Write, conf: &Config) -> Result<()> {
...
+// DO NOT EDIT"));
+
+    Ok(())
+}
+
+pub fn single_version_file(w: &mut Write, conf: &Config) -> Result<()> {
+    try!(writeln!(
+        w,
+        "All files in this directory
+was generated by gir (https://github.com/gtk-rs/gir @ {})
```
Comment (GuillaumeGomez) [inline src/codegen/general.rs]: "was" -> "were"

File: README.md (hunk 1 above)
Comment (EPashkin) [reply README.md]: I want change it do default after some check

**Comment by GuillaumeGomez** [comment]
Just a few nits but otherwise seems good to me.

File: README.md (hunk 1 above)
Comment (GuillaumeGomez) [reply README.md]: Fine by me!

File: src/codegen/general.rs (hunk 2 above)
Comment (EPashkin) [reply src/codegen/general.rs]: Thanks

File: src/codegen/general.rs [hunk 3]
```diff
@@ -24,6 +27,29 @@ pub fn start_comments(w: &mut Write, conf: &Config) -> Result<()> {
     Ok(())
 }
 
+pub fn start_comments_no_version(w: &mut Write) -> Result<()> {
+    try!(writeln!(
+        w,
+        "// This file was generated by gir (https://github.com/gtk-rs/gir)
+// DO NOT EDIT"));
```
Comment (GuillaumeGomez) [inline src/codegen/general.rs]: Can you add `from gir-files (https://github.com/gtk-rs/gir-files)` as well please?

Comment (EPashkin) [reply src/codegen/general.rs]: Done

**Comment by GuillaumeGomez** [comment]
Sounds all good for me now. Just waiting for @sdroege's confirmation. :)

**Comment by sdroege** [comment]
Seems good to me but I don't like the filename :) Why does it start with an underscore? `README.TXT` maybe?

**Comment by EPashkin** [comment]
Underscore to make file first in directory. README.TXT will be last, so also good, but I personally don't like uppercase in names.
@GuillaumeGomez Which name you prefer?

**Comment by GuillaumeGomez** [comment]
What about `versions.txt`? At least it's clear what this file is about.

**Comment by EPashkin** [comment]
Ok, I change to `versions.txt`

**Pull request merged**